composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "sminnee/callbacklist",
+    "name": "sminnee\/callbacklist",
```

Comment:
Why did you do this? It doesn't seem necessary and it weakens readability. Escaping a backslash is fine but not a forward slash.

Reply:
23:40 my time (UK), not going to debug now, but I suspect a side effect of the tool I wrote to add the continuous integration tools

Reply:
I agree that it is icky